### PR TITLE
Add Louisiana EITC and main rates

### LINKED ISSUE #2035: Louisiana EITC
Refundable
5% of federal since 2019
3.5% at some point before then

https://www.legis.la.gov/legis/Law.aspx?d=453085

### PR DESCRIPTION
**Pull request by MaxGhenis**
Fixes #2035

### CODE REVIEW

File: policyengine_us/variables/gov/states/la/la_eitc.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.legis.la.gov/legis/Law.aspx?d=453085"
+    defined_for = StateCode.LA
+
+    def formula(tax_unit, period, parameters):
+        federal_eitc = tax_unit("earned_income_tax_credit", period)
+        match_percent = parameters(
+            period
+        ).gov.states.co.tax.income.credits.eitc.match
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/la/la_eitc.py:17]:
```suggestion
        ).gov.states.la.tax.income.credits.eitc.match
```

File: policyengine_us/variables/gov/states/la/la_eitc.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class la_eitc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "LA EITC"
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/la/la_eitc.py:7]:
```suggestion
    label = "Louisiana EITC"
```

File: policyengine_us/parameters/gov/states/la/rates
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/la/rates:1]: delete file (not rates.yaml)

File: policyengine_us/tests/policy/baseline/gov/states/la/tax/income/la_income_tax_before_credits.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+- name: income in 2nd tax bracket
+  period: 2022
+  absolute_error_margin: 1
+  input:
+    la_taxable_income: 40_000
+    state_code: LA
+  output:
+    # 12_500*0.0185+(40_000-12_500)*0.0315=1097.5
+    la_income_tax_before_credits: 1097.5
```
Comment (MaxGhenis) [inline policyengine_us/tests/policy/baseline/gov/states/la/tax/income/la_income_tax_before_credits.yaml:19]:
```suggestion
    la_income_tax_before_credits: 1_097.5
```

File: policyengine_us/tests/policy/baseline/gov/states/la/tax/income/la_income_tax_before_credits.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- name: income in 3rd tax bracket
+  period: 2022
+  absolute_error_margin: 1
+  input:
+    la_taxable_income: 60_000
+    state_code: LA
+  output:
+    #12_500*0.0185+(50_000-12_500)*0.0315+(60_000-50_000)*0.0425=1837.5
+    la_income_tax_before_credits: 1837.5
```
Comment (MaxGhenis) [inline policyengine_us/tests/policy/baseline/gov/states/la/tax/income/la_income_tax_before_credits.yaml:29]:
```suggestion
    la_income_tax_before_credits: 1_837.5
```

**Review by MaxGhenis — COMMENTED** [review]
* Add `changelog_entry.yaml` (see other PRs)
* Run `git pull upstream master`
* Run `make format`

File: policyengine_us/parameters/gov/states/la/match.yaml
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/la/match.yaml:1]: move these into `states/la/tax/income/credits/eitc/match.yaml` and `states/la/tax/income/rates.yaml`

File: policyengine_us/parameters/gov/states/la/rates.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+     2022-01-01: 0.0315
+ - threshold:
+     2022-01-01: 50_000
+   rate:
+     2022-01-01: 0.0425
+
+
+metadata:
+  reference:
+
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/la/rates.yaml:20]:
```suggestion
```

File: changelog_entry.yaml [hunk 6]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    changed:
```
Comment (MaxGhenis) [inline changelog_entry.yaml:3]:
```suggestion
    added:
      - Louisiana EITC.
      - Louisiana main rates.
```

**Review by MaxGhenis — COMMENTED** [review]
also run `make format` and `git pull upstream master` (in separate commits)

**Comment by PavelMakarchuk** [comment]
Duplicate PR ; new : #2358

**Pull request closed without merging**